molgenis/molgenis-emx2
# fix(directory): make sure the filter is a array before mapping it

What are the main changes you did:
- `Collaborationtype` filter returns a boolean value where the `getHumanReadableString` function expects an array this case is now handled

how to test:
- With the "Available to commercial use" filter set, it should now not show a console error on when pressing "Send to the Negotiator"
